execution-engine/src/fs.rs

```diff
@@ -14,19 +14,18 @@
 //! See the `LICENSE_MIT.markdown` file in the Veracruz root directory for
 //! information on licensing and copyright.
 
+use log::info;
 use platform_services::getrandom;
 use std::{
     collections::HashMap,
     convert::AsRef,
-    path::{Path, PathBuf},
+    ffi::OsString,
+    os::unix::ffi::{OsStrExt, OsStringExt},
+    path::{Component, Path, PathBuf},
     vec::Vec,
 };
 use veracruz_utils::policy::principal::{Principal, RightsTable, StandardStream};
-use wasi_types::{
-    Advice, DirCookie, DirEnt, ErrNo, Event, Fd, FdFlags, FdStat, FileDelta, FileSize, FileStat,
-    FileType, Inode, LookupFlags, OpenFlags, PreopenType, Prestat, RiFlags, Rights, RoFlags,
-    SdFlags, SetTimeFlags, SiFlags, Size, Subscription, Timestamp, Whence,
-};
+use wasi_types::*;
```

Comment:
I think we should avoid these types of wildcard import, in favour of being explicit about what it is we are importing from a particular library.